src/containers/profiles/store/profileActions.js

```diff
@@ -0,0 +1,47 @@
+
+import api from '../../../utils/api';
+import {
+  VIEW_USER_PROFILE, USER_PROFILE_ERROR, EDIT_USER_PROFILE, EDIT_PROFILE_ERROR,
+} from './types';
+
+// const apiUrl = 'http://localhost:8000/api/profiles/';
+// const token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MSwidXNlcm5hbWUiOiJiZXZrb2xvbGkxIiwiZW1haWwiOiJrb2xvbGliZXZhaEBnbWFpbC5jb20iLCJpYXQiOjE1NDE1NjQxNzYsImV4cCI6MTU0MTc4MDE3Nn0.S2E6sdE1SvL0LF9TbvKwt02RV3OF6aQc6yGfCbbpXjY';
+// const config = {
+//   headers: {
+//     Authorization: `Token ${token}`,
+//     'Content-Type': 'multipart/form-data',
+//     Accept: 'application/json',
+//   },
+// };
+
+export const getUserProfileAction = username => (dispatch) => {
+  api
+    .get(`profiles/${username}/`)
+    .then((data) => {
+      dispatch({
+        type: VIEW_USER_PROFILE,
+        payload: data,
+      });
+    }).catch((errors) => {
+      dispatch({
+        type: USER_PROFILE_ERROR,
+        payload: errors,
+      });
+    });
+};
+
+export const editUserProfileAction = Editdata => (dispatch) => {
+  api
+    .put('profiles/bevkololi1/', Editdata)
```

Comment:
Hey @bevkololi ,
As I commented on one of the files, maybe you could work on the `editUserProfileAction` to also be dynamic instead of passing the hard-coded endpoint `profiles/bevkololi1` to reflect something like `'profiles/${username}/'`. I'm sure that what you intend to do but it's good I point it out 💯

Reply:
+1

Reply:
Thanks @SrMoffat  noted